.github/workflows/e2e-node.yml

```diff
@@ -0,0 +1,39 @@
+name: End-to-end node tests
+
+on:
+  push:
+
+env:
+  CI: true
+jobs:
+  e2e-node:
+    # Dependabot cannot access secrets, so it doesn't have a token to authenticate to ESS.
+    # Since all the other jobs of this workflow depend on this one, skipping it should
+    # skip the entire workflow.
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    runs-on: ${{ matrix.os }}
+    environment:
+      name: ${{ matrix.environment-name }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [16.x]
+        # NSS does not support static client registration, which we rely on for testing.
+        environment-name: ["ESS Dev-Next"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: npm
+      - run: npm ci
+      - run: npm run build
```

Comment:
I don't think the build step here is required (and if not, it should be removed from other CI scripts as well, because it's definitely in other repos)

Reply:
it's only needed because the build isn't carried through the pipeline, this is where we need to merge the pipelines and make the `dist` output an artifact.

Reply:
Aaah, yeah, true — arguably it _should_